### PR TITLE
Add Chinese translation citation

### LINKED ISSUE #65: Translators missing on the citation page
I noticed that on [the citation page](https://agda-zh.github.io/PLFA-zh/Citing/), the references and the BibTeX do not include the translators. I feel you should add these! This might be as simple as adding translators in the appropriate places, as I've done below, but it probably makes sense to translate the titles in their entirety.

> Wadler, Philip and Wen Kokke. Programming Language Foundations in Agda. Translated by Fangyi Zhou, Oling Cat, and Rongxiao Fu. Available at https://agda-zh.github.io/PLFA-zh/. 2019.

```bibtex
@Book{plfa2019,
  author     = {Philip Wadler and Wen Kokke},
  title      = {Programming Language Foundations in {A}gda},
  translator = {Fangyi Zhou and Oling Cat and and Rongxiao Fu},
  note       = {Available at \url{https://agda-zh.github.io/PLFA-zh/}},
  year       = 2019,
}
```

### PR DESCRIPTION
**Pull request by OlingCat**
Fixed #65

### CODE REVIEW

**Comment by OlingCat** [comment]
参考了下知网的引用格式，不过还是不太确定。

File: src/plfa/citing.md [hunk 1]
```diff
@@ -6,17 +6,38 @@ permalink      : /Citing/
...
+
 > Wadler, Philip and Wen Kokke.
 > _Programming Language Foundations in Agda_.
 > Available at `http://plfa.inf.ed.ac.uk`.
 > 2019.
 
+中文版：
+
+> Philip Wadler, Wen Kokke.
+> 编程语言基础：Agda 语言描述[M].
```
Comment (fangyi-zhou) [inline src/plfa/citing.md]: `[M]` 是什么

File: src/plfa/citing.md [hunk 2]
```diff
@@ -6,17 +6,38 @@ permalink      : /Citing/
...
 	@Book{plfa2019,
 	  author = {Philip Wadler and Wen Kokke},
 	  title  = {Programming Language Foundations in {A}gda},
 	  note   = {Available at \url{http://plfa.inf.ed.ac.uk/}},
 	  year   = 2019,
 	}
+
+中文版：
+
+	@Book{plfa2019,
```
Comment (fangyi-zhou) [inline src/plfa/citing.md]:
```suggestion
	@Book{plfazh2019,
```

File: src/plfa/citing.md (hunk 1 above)
Comment (OlingCat) [reply src/plfa/citing.md]: [M] 表示书籍，专著

File: src/plfa/citing.md [hunk 3]
```diff
@@ -6,17 +6,38 @@ permalink      : /Citing/
...
+
+中文版：
+
+	@Book{plfazh2019,
+	  author     = {Philip Wadler and Wen Kokke},
+	  title      = {编程语言基础：{A}gda 语言描述},
+	  translator = {周方易 and 欧林猫 and 傅荣枭},
+	  note       = {可访问 \url{https://agda-zh.github.io/PLFA-zh/} 阅读},
+	  year       = 2019,
+	}
```
Comment (ice1000) [inline src/plfa/citing.md]: Missing `translator`s. See https://tex.stackexchange.com/a/19709/145304

Comment (fangyi-zhou) [reply src/plfa/citing.md]: I don't understand what you mean

Comment (ice1000) [reply src/plfa/citing.md]: I was blind

**Pull request merged**